vic.api.v0.yaml

```
@@ -486,6 +489,57 @@ paths:
           $ref: '#/components/responses/VendorGroupDeletedResponse'
         default:
           $ref: '#/components/responses/ErrorResponse'
+  /attachments:
+    post:
+      description: |
+        Use this to upload an attachment to Vic.ai. The attachment will be
+        created and processing enqueued automatically. The response is
+        a 201 with the attachment ID.
+        Supported content types are:
+        - application/msword
+        - application/pdf
+        - application/vnd.ms-word.document.macroEnabled.12
+        - application/vnd.ms-word.template.macroEnabled.12
+        - application/vnd.openxmlformats-officedocument.wordprocessingml.document
+        - application/vnd.openxmlformats-officedocument.wordprocessingml.template
+        - image/tiff
+        - `text/xml`
```

Why not just `text/xml`? 🤔 

```suggestion
        - text/xml
```

---

weird, thanks 👍